plugins/linear-comparative-view/src/BreakpointSplitRenderer/BreakpointSplitRenderer.ts

```diff
@@ -194,9 +169,6 @@ export default class BreakpointSplitRenderer extends ComparativeServerSideRender
         //   continue
         // }
 
-        // flipMultiplier combines with normal directionality of the curve
-        const flipMultipliers = views.map(v => 1)
-
```

Comment:
@cmdcolin since all values in `flipMultipliers` were `1`, it seemed like it could be removed, but I can put it back if needed.

Reply:
breakpoint split renderer is actually currently unused. I'd say it's fine to remove, it might be restored, but probably it would be restored in a different way